### PR TITLE
Added basic server

### PR DESCRIPTION
**Pull request by b11n**
Added initial routes for the application.

PTAL if the file structure and code structure looks good.

### CODE REVIEW

File: server/main.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+type event struct {
```
Comment (neetupm13) [inline server/main.go]: move this to a different package, also make this exportable

File: server/main.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	ID    string `json:"id"`
+	Title string `json:"title"`
+	From  string `json:"from"`
```
Comment (neetupm13) [inline server/main.go]: why not use Time type for From and To?

File: server/main.go (hunk 1 above)
Comment (b11n) [reply server/main.go]: Done

File: server/main.go (hunk 2 above)
Comment (b11n) [reply server/main.go]: Done

**Review by neetupm13 — APPROVED** [review]

**Pull request merged**